File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        first_index = 0
        for first in nums:
            second_index = 0
            for second in nums[1:]:
                second_index += 1
                if first_index == second_index:
                    continue
                if target == first + second:
                    return [nums.index(first), second_index]
            first_index += 1

File: 0001-two-sum/0001-two-sum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {
    int i;
    int j;
    int *new;

    new = malloc(sizeof(int) * 2);
    if (!new)
    return NULL;
    i = 0;

    while (i < numsSize)
    {
        j = 0;
        while (j < numsSize)
        {
            if (j == i)
            {
                j++;
            }
            if (nums[i] + nums[j] == target)
            {
                new[0] = i;
                new[1] = j;
                *returnSize = 2;
                return new;
            }
            j++;
        }
        i++;
    }
    return NULL;
}